src/api/app/assets/stylesheets/webui/notifications.scss

```diff
@@ -1,10 +1,11 @@
 $notifications-filter-box-height: 3.5rem;
+$avatars-counter-size: 1.4375rem;
```

Comment:
We want to reuse this in the timeline CSS instead of having what appears to be a magic number.